src/libstd/process.rs

```diff
@@ -1625,6 +1625,17 @@ impl<E: fmt::Debug> Termination for Result<(), E> {
     }
 }
 
+#[unstable(feature = "termination_trait_lib", issue = "43301")]
+impl Termination for Option<()> {
```

Comment:
why only `()` rather than anything implementing `Termination`?

Reply:
Would that flatten the `Option`? I'm afraid this could lead to too clever code like `fn main -> Option<Option<Result<(), _>>> { .. }`.